### PR TITLE
lerna, yarn packages init

### PR DESCRIPTION
**Pull request by lynchbomb**
Initial move to mono-repo structure.

### CODE REVIEW

**Comment by runspired** [comment]
@lynchbomb is there a reason for the strict copy vs move for most of these files? move will preserve hit history.

**Comment by lynchbomb** [comment]
@runspired yeah something wonky is happening with the mono-repo config. Hang-tight on this PR.

**Comment by lynchbomb** [comment]
@runspired figured out the issue with Travis. Which was related to the pathing change to `node_modules` for the rollup build files now that we are a mono-repo.

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-  - "6"
+  - "lts/*"
   - "node"
 
 cache: yarn
 before_install:
-  - nvm install 7
-  - nvm use 7
+  - nvm install --lts
+  - nvm use --lts
```
Comment (runspired) [inline .travis.yml]: this is awesome, had no idea this was possible

**Pull request merged**